lib/diagnostic.rb

```diff
@@ -8,7 +8,11 @@
 # and location should only be writable.
 
 ##
-# your code here
+class Person
+  attr_accessor :name
+  attr_reader :age
+  attr_writer :location
+end
```

Comment:
Halfway there. This doesn't satisfy the direction "All three attributes should be defined when you instantiate a new Person.". To do that, you'll need to define an `initialize